slick.groupitemmetadataprovider.js

```diff
@@ -155,7 +155,7 @@
       return {
         selectable: false,
         focusable: options.totalsFocusable,
-        cssClasses: options.totalsCssClass,
+        cssClasses: options.totalsCssClass + ' slick-group-level-' + item.group.level,
```

Comment:
can you add a variable `var groupLevel = item && item.group && item.group.level;` and use it in the `cssClasses` just to be on the safe side. I'm getting used to have TypeScript to very strict mode and that is how I would code it. You can do the same on the other line

Reply:
Change made, thanks!